includes/class-edd-fees.php

```diff
@@ -107,6 +107,9 @@ public function add_fee( $args = array() ) {
 		// Sanitize the amount
 		$args['amount'] = edd_sanitize_amount( $args['amount'] );
 
+		// Force the amount to have 2 decimal places
+		$args['amount'] = number_format( (float) $args['amount'], 2, '.', '' );
```

Comment:
@sunnyratilal By using that then we can also write some unit tests around other decimal precisions as well.